src/main/java/com/entropyteam/entropay/employees/dtos/EmployeeDto.java

```diff
@@ -5,22 +5,26 @@
 import java.util.List;
 import java.util.UUID;
 import java.util.stream.Collectors;
+
+import com.entropyteam.entropay.employees.models.*;
```

Comment:
please revert this change. It is not a good practice to use `*` in the imports.

Reply:
Reverted